lib/grandpa/vote_message.go

```diff
@@ -126,51 +126,67 @@ func (s *Service) validateVoteMessage(from peer.ID, m *VoteMessage) (*Vote, erro
 	// check for message signature
 	pk, err := ed25519.NewPublicKey(m.Message.AuthorityID[:])
 	if err != nil {
+		// TODO: affect peer reputation
```

Comment:
This we need to add logic for a reputation change? Should there be a corresponding issue for this?

Reply:
@jimjbrettj yeah that's the two checkboxes I have in the PR description that are, well, still unchecked :smile: But good spot nonetheless!

@danforbes please let me know if you plan on creating the issues I mentioned to you on Slack, otherwise let me know and I can create them :+1: So I can link them in the comments in this PR.

Reply:
I pushed a commit with comments linking to the grandpa politeness issue now #2505